src/Features/CSharp/Portable/CSharpFeaturesResources.Designer.cs

```diff
@@ -548,7 +548,16 @@ internal class CSharpFeaturesResources {
             }
         }
         
-         /// <summary>
+        /// <summary>
+        ///   Looks up a localized string similar to Invert if statement.
+        /// </summary>
+        internal static string Invert_if_statement {
+            get {
+                return ResourceManager.GetString("Invert_if_statement", resourceCulture);
```

Comment:
Note: based on current discussions around namign the "Convert 'for/query/foreach'" features, this should probably just be "Invert 'if'" (and "Invert 'If'" for VB).

Reply:
but i don't feel strongly.